spec.md

```diff
@@ -1811,6 +1811,7 @@ expressed in the short form.
 - `read_only`: flag to set the volume as read-only
 - `bind`: configure additional bind options
   - `propagation`: the propagation mode used for the bind
+  - `create_host_path`: create source path on host if missing. This is automatically implied by short syntax.
```

Comment:
Nit: document this is caused by legacy reasons ?

Reply:
```suggestion
  - `create_host_path`: create a directory at the source path on host if there is nothing present. Do nothing if there is something present at the path. This is automatically implied by short syntax.
```